bolt-http4k/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
 
     <properties>
-        <http4k.version>4.4.1.0</http4k.version>
+        <http4k.version>4.5.0.1</http4k.version>
```

Comment:
As http4k very often publishes new versions, we always built with the latest version (except major releases)